modules/vulnerability-management20-permissions.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_content-type: CONCEPT
+[id="vulnerability-management20-permissions_{context}"]
+= Vulnerability report permissions
+
+[role="_abstract"]
+The ability to create, view, and download reports depends on the access scope for your user account.
+
+For example, you can only view, create, and download reports for data that your user account has permission to access. In addition, the following restrictions apply:
```

Comment:
Are you referring to `Access` as shown in access control section of the product?

Reply:
I rewrote this to try and clear up confusion, can you tell me if it's accurate?

Reply:
Recent version is clear